README.md

```diff
@@ -82,6 +82,12 @@ Changes in an LTS-covered major version are limited to:
 5. Certain performance improvements where the risk of breaking existing applications is minimal;
 6. Changes that introduce large amount of code churn where the risk of breaking existing applications is low and where the change in question may significantly ease the ability to backport future changes due to the reduction in diff noise.
 
+If a change updates documentation or tests, or resolves a breaking regression,
```

Comment:
@sam-github upon further thought, `breaking regression`, does not sound correct, and may not be meaningful. The only example of a non-breaking regression I could think of would be a regression introduced by a patch or minor change that removed documentation that had existed in an LTS release. That would be non-breaking in the sense that processes dependent on Node would continue operating, while there was a regression in the product shipped.